deploy/zfs-operator.yaml

```diff
@@ -47,8 +47,450 @@ spec:
     name: Filesystem
     description: filesystem created on the volume
     type: string
+---
+##############################################
+###########                       ############
+###########   Snapshot CRDs       ############
+###########                       ############
+##############################################
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: zfssnapshots.openebs.io
+spec:
+  group: openebs.io
+  version: v1alpha1
+  scope: Namespaced
+  names:
+    plural: zfssnapshots
+    singular: zfssnapshot
+    kind: ZFSSnapshot
+    shortNames:
+    - snapshot
```

Comment:
can shortname be zsnap or zsnapshot - to avoid being confused with regular snapshots?

Reply:
changed them to zfssnapshot, zfssnap